scripts/do-release.sh

```diff
@@ -107,8 +111,22 @@ function release_images() {
 }
 
 function tag_images() {
-    # Creating a local tag so that images are uploaded with it
-    _git tag -a -f "${release['version']}" -m "${release['version']}"
+    # Tag the images matching the release commit using the release tag
+    local project_version
+    project_version=$(cd "projects/${project}" && make print-version BASE_BRANCH="${release['branch']:-devel}" | \
+                      grep -oP "(?<=CALCULATED_VERSION=).+")
+    local hash="${project_version#v}"
+    
+    # Log in without dryrun to make sure the credentials are OK
+    echo "$QUAY_PASSWORD" | skopeo login quay.io -u "$QUAY_USERNAME" --password-stdin
```

Comment:
This has to be in `dryrun` mode since it's used for testing, and at that point we don't want any interactions that require authorization (at least we try to minimize them as much as possible)

Reply:
Perhaps we should rely on login being done externally, as is currently the case for `docker push`.

Reply:
sure though that sounds reasonable as an improvement